.github/CONTRIBUTING.md

```diff
@@ -25,8 +25,14 @@ Please see our [wiki article](https://ardupilot.org/dev/docs/submitting-patches-
 
 The ArduPilot project is open source and [maintained](https://github.com/ArduPilot/ardupilot#maintainers) by a team of volunteers.
 
-To contribute, you can send a pull request on Github. You can also join the
+To contribute, you can send a pull request on GitHub.
+
+New developers are recommended to join the `#general` channel on
+[Discord](https://ardupilot.org/discord), which is suitable for
+asking both development and non-development questions. :smile:
```

Comment:
Thanks for this, I'm not sure that we're encouraging non-developer questions on Discord though.  Maybe just remove the part that starts with "which is suitable .."

Reply:
No worries.  Update coming...